cypress/integration/query/search_query_spec.js

```diff
@@ -0,0 +1,24 @@
+describe("Search Query", () => {
```

Comment:
Once I see a _failing_ build for this PR, I'm going to merge this so I can test #3631. But please do take a moment to review it and suggest improvements if needed.